### PR TITLE
Fix blank title for authenticator

### PR DESCRIPTION
**Pull request by dshokouhi**
## Summary

Fixes the following sentry error by directly setting the title in the method, this seems similar to the blank notification title we saw with the `Sensor Worker` so the fix is the same:

```
java.lang.IllegalArgumentException: Title must be set and non-empty.
    at androidx.biometric.BiometricPrompt$PromptInfo$Builder.build(BiometricPrompt.java:570)
    at io.homeassistant.companion.android.authenticator.Authenticator.authenticate(Authenticator.kt:46)
    at io.homeassistant.companion.android.webview.WebViewActivity.onWindowFocusChanged(WebViewActivity.kt:740)
    at androidx.appcompat.view.WindowCallbackWrapper.onWindowFocusChanged(WindowCallbackWrapper.java:124)
    at com.android.internal.policy.DecorView.onWindowFocusChanged(DecorView.java:2319)
    at android.view.View.dispatchWindowFocusChanged(View.java:15472)
    at android.view.ViewGroup.dispatchWindowFocusChanged(ViewGroup.java:1520)
    at android.view.ViewRootImpl.handleWindowFocusChanged(ViewRootImpl.java:3926)
    at android.view.ViewRootImpl.access$1400(ViewRootImpl.java:250)
    at android.view.ViewRootImpl$ViewRootHandler.handleMessage(ViewRootImpl.java:6026)
    at android.os.Handler.dispatchMessage(Handler.java:106)
    at android.os.Looper.loop(Looper.java:246)
    at android.app.ActivityThread.main(ActivityThread.java:8425)
    at java.lang.reflect.Method.invoke(Method.java)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:596)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
```

## Link to pull request in Documentation repository

Documentation: home-assistant/companion.home-assistant#

### CODE REVIEW

File: app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt [hunk 1]
```diff
@@ -745,7 +745,7 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
             if (presenter.isLockEnabled() && !unlocked)
                 if ((System.currentTimeMillis() > presenter.getSessionExpireMillis())) {
                     blurView.setBlurEnabled(true)
-                    authenticator.authenticate()
+                    authenticator.authenticate(applicationContext.getString(R.string.biometric_title))
```
Comment (nick2525) [inline app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt:748]: authenticator.authenticate(getString(R.string.biometric_title)) as getString() is available at fragment

File: app/src/main/java/io/homeassistant/companion/android/settings/SettingsFragment.kt [hunk 2]
```diff
@@ -92,8 +92,7 @@ class SettingsFragment : PreferenceFragmentCompat(), SettingsView {
                 isValid = true
                 if (BiometricManager.from(requireActivity()).canAuthenticate() == BiometricManager.BIOMETRIC_SUCCESS) {
                     setLock = true
-                    authenticator.title = getString(R.string.biometric_set_title)
-                    authenticator.authenticate()
+                    authenticator.authenticate(requireContext().getString(R.string.biometric_set_title))
```
Comment (nick2525) [inline app/src/main/java/io/homeassistant/companion/android/settings/SettingsFragment.kt:95]: authenticator.authenticate(getString(R.string.biometric_set_title)) as getString() is available at fragment

File: app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt (hunk 1 above)
Comment (dshokouhi) [reply app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt:748]: ah good catch, force of habit here :)

File: app/src/main/java/io/homeassistant/companion/android/settings/SettingsFragment.kt (hunk 2 above)
Comment (dshokouhi) [reply app/src/main/java/io/homeassistant/companion/android/settings/SettingsFragment.kt:95]: ah good catch, force of habit here :)

**Pull request merged**